src/extension/import-grouping/ImportGroupSettingParser.ts

```diff
@@ -11,10 +11,12 @@ import { RemainImportGroup } from './RemainImportGroup';
  */
 export type ImportGroupSetting = string | { identifier: string, order: ImportGroupOrder };
 
+const REGEX_REGEX_GROUP = /^\/.+\/$/;
```

Comment:
- `.+.*` looks like a RegexDOS invite. There's no need at all to add `.*` at the end here.
- Stripping useless costly `g` flag